test/Benchmarks/src/Column_Numeric.enso

```diff
@@ -6,7 +6,7 @@ from Standard.Test import Bench, Faker
 ## Bench Utilities ============================================================
 
 vector_size = 1000000
-iter_size = 100
+iter_size = 10
```

Comment:
should we leave this at 100? or is it so long now we want it shorter!

Reply:
It takes 47s with `iter_size = 10` so I assume we don't want it much longer than that.

Reply:
For future - I think in that case we may want to consider to reduce the `vector_size` instead, to make each iteration a bit shorter - but to keep the number of iterations. 10 may be too few to see warmup and also to see the distribution of the timings (i.e. if the variance is low or high).